src/Exception/InvalidVersionNumber.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\ContentApiBundle\Exception;
+
+use Throwable;
+use UnexpectedValueException;
+
+class InvalidVersionNumber extends UnexpectedValueException
```

Comment:
could be `InvalidArgumentException`, `DomainException`... the utility of the base exception types is limited given they are so generic and sometimes overlapping